poll/async/get.js

```diff
@@ -80,3 +85,16 @@ function decoratedPoll (rawPoll, msgs = []) {
 
   return poll
 }
+
+function decoratePosition ({position: rawPosition, poll: rawPoll}) {
+  var position = getMsgContent(rawPosition)
+  var poll = getMsgContent(rawPoll)
+
+  var newPosition = Object.assign({}, rawPosition)
+
+  if (isChooseOnePoll(poll)) {
```

Comment:
I'm sure there is a better way than this.

Reply:
heads up this won't work because isChooseOnePoll is an injected one.
I would use 
```js
const isPoll = require('../../isPoll')

isPoll.chooseOne(poll)
```

Reply:
I know what you mean about the pivoting. I think we just need to write verbose and see how it pans out

Reply:
speaking of pivoting, I think we could  add poll type as a decoration `poll.pollDetails.type` OR `position.positionDetails.type`

useful given on the `newPosition` you'd have to otherwise go `newPosition.value.content.positionDetails.type`

Reply:
:white_check_mark:  changed this

Reply:
:white_check_mark: added this (type thing)

Reply:
actually I'm stupid, it's already on the decorated poll...